packages/react-vis/tests/components/parallel-coordinates-tests.js

```diff
@@ -74,15 +80,16 @@ test('Parallel Coordinates: Animated Parallel Coordinates ', t => {
     'should find the right number of axes'
   );
   t.equal(
-    $.find('.rv-parallel-coordinates-chart-line').length,
+    $.find('path.rv-parallel-coordinates-chart-line').length,
```

Comment:
Is there a lint rule catching these? Otherwise, might be better to leave it as before since the element/tagname is an implementation detail.

Reply:
These are causing the tests to fail.
If we want to better hide the implementation details, then we should probably switch to `shallow` rendering.
I've never been a fan of `mount` in tests as it causes way more rendering and logic to be executed than I generally want to test.